pkg/adapter/cronjobevents/adapter.go

```diff
@@ -87,13 +83,17 @@ func (a *Adapter) Start(ctx context.Context, stopCh <-chan struct{}) error {
 func (a *Adapter) cronTick() {
 	logger := logging.FromContext(context.TODO())
 
+	eventContext := cloudevents.EventContextV02{
+		Type:   sourcesv1alpha1.CronJobEventType,
+		Source: *types.ParseURLRef(sourcesv1alpha1.CronJobEventSource),
+	}.AsV02()
+	eventContext.SetSubject(a.Schedule)
```

Comment:
@grantr Would this be a problem for CEL-based filtering? That is, subject would have things like "*/1 * * * *"

Reply:
That particular subject seems not very useful. I'd expect there to be something that's more meaningful for the subscriber, say, name. I could have two hourly jobs: backup and logshred, or something like that, they would both have the same schedule.
Thoughts?

Reply:
You are right. Name would be more useful here. Will update that. 
Thanks for the comment

Reply:
Done!